azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -121,7 +121,17 @@
    * default one the system should pre-fetch hcat token from. Note: Multiple thrift uris are
    * supported, use comma to separate the values, values are case insensitive.
    */
+  // Use EXTRA_HCAT_CLUSTERS instead
+  @Deprecated
   private static final String EXTRA_HCAT_LOCATION = "other_hcat_location";
+  /**
+   * this parameter is used to replace EXTRA_HCAT_LOCATION that could fail when one of the uris is not available.
+   * EXTRA_HCAT_CLUSTERS has the following format:
+   * other_hcat_clusters = "(thrift://hcat1:port,thrift://hcat2:port),(thrift://hcat3:port,thrift://hcat4:port)"
+   * Each string in the parenthesis is regarded as a "cluster", which we traverse until we get a delegation token
```

Comment:
The comment is misleading; the code should not stop when it gets a delegation token from one cluster; it should get delegation tokens from all clusters.